src/libs/ReimbursementAccountUtils.js

```diff
@@ -18,44 +17,55 @@ function getDefaultStateForField(props, fieldName, defaultValue = '') {
 }
 
 /**
- * @param {Object} props
+ * Use this function binding it to a component's instance
```

Comment:
IMO we have made this method less flexible and harder to test since it now requires a "component instance" to be bound to it.

Reply:
Hmm I thought I could be making it harder to test, but then I thought that in a test you would have to bind it to an object, not necessarily a component and that shouldn't be too complicated.

When I changed the errors to be stored in the state of the component, I started needing access to `state` and `setState` instead of just `props`. I thought that to pass `setState` I would need to bind `setState` to `this` anyway before passing it, so it made some sense to me to make these functions "binded" for easier use in the components. Not all of the modified functions benefited to be like this, but I changed them all in the same way to make them more consistent.